src/pages/playground/sidebar.astro

```diff
@@ -3,7 +3,7 @@ import SidebarItem from '@/atoms/sidebar-item.astro';
 import Sidebar from '@/organisms/sidebar.astro';
 ---
 
-<Sidebar>
+<Sidebar className="">
```

Comment:
Jak tutaj potrzebowałeś sidebara bez klasy to luz — możesz ją zrobić opcjonalną. Wcześniej musiałem przeoczyć ten use case 😅